proc_dash/app.py

```diff
@@ -5,12 +5,12 @@
 
 import dash_bootstrap_components as dbc
 import pandas as pd
+from dash.dependencies import Input, Output, State
```

Comment:
Ah, this was an automatic change made by `isort` in `pre-commit`,

Reply:
Ah, this was an automatic change made by `isort` in `pre-commit`. Interestingly, I dug into this a little bit and realized the reason it didn't move the `from dash import ALL ...` line is because I had renamed my local directory for the repo to be `dash`. So I think isort got confused and thought this line was an application-specific import 👀 instead of a third-party import.

I renamed my repo directory to `proc_dash` and it re-ordered the imports to have all the `dash` imports together.

Good catch!